packages/custom_lint_core/lib/src/configs.dart

```diff
@@ -35,13 +39,28 @@ class CustomLintConfigs {
     final include = yaml['include'] as Object?;
     var includedOptions = CustomLintConfigs.empty;
     if (include is String) {
-      final includeAbsolutePath = absolute(
-        analysisOptionsFile.parent.path,
-        include,
-      );
-      includedOptions = CustomLintConfigs.parse(
-        analysisOptionsFile.provider.getFile(includeAbsolutePath),
-      );
+      final includeUri = Uri.parse(include);
+      String? includeAbsolutePath;
+
+      if (include.startsWith('package:')) {
+        final packageConfig = parsePackageConfigSync(Directory.current);
```

Comment:
This would lead to a recursive parsing of the same file.
We should likely pass this as parameter.

IMO it makes sense to parse it from custom_lint_builder's `client.dart#_ClientAnalyzerPlugin.afterNewContextCollection` and pass it along

Reply:
I've changed this as you suggested and added the `packageConfig` parameter to the `parse` factory method. Please check the current implementation and leave your feedback. 
Thank you in advance!